### DISCUSSION TITLE
Github.io page custom domain redirecting to branded.me

### DISCUSSION THREAD

**Discussion by bgschust** [community/community#43261] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi I recently set up my github.io pages to point to cropify.org and www.cropify.org. I added www.cropify.org in the custom domains section of the settings, and I updated the A records and CNAME record in Bluehost, where my domain is hosted. There is also a CNAME file in the repository that simply says "www.cropify.org".

My DNS records in bluehost are as follows:
![image](https://user-images.githubusercontent.com/61630536/210460215-b811ca14-45be-40b9-a8fe-0f1a05c1e276.png)

Do you have any ideas why my base domains are pointing to a random website, branded.me, and reloading over and over, while the full URL cropify.org/cropify/index.html goes to the right place?

**Comment by CodesdaLu** [comment]
Hi, 
so to direct to cropify you can create an html link and put https://www.cropify.org/cropify/general-items.html.

if you are going to create github pages(site) you need to define the name of the repository and follow these steps

https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site

if your question is answered please mark it as answered
